jazzmin/static/jazzmin/css/main.css

```diff
@@ -829,3 +829,9 @@ body.jazzmin-login-page {
 .callout {
     color: black;
 }
+
+/* tweeks for django-filer*/
+#content-main {
```

Comment:
Can this be targeted to django-filer somehow ? are there any classes added to parent elements that will stop this affecting other `#content-main` elements  as its a fairly common id for a container

Reply:
@farridav I change this to more specific selector `.navigator-top-nav + #content-main`. Child element of content-main has `width: 100%;`.